airflow/cli/commands/remote_commands/task_command.py

```diff
@@ -231,7 +231,13 @@ def task_failed_deps(args) -> None:
     Trigger Rule: Task's trigger rule 'all_success' requires all upstream tasks
     to have succeeded, but found 1 non-success(es).
     """
-    dag = get_dag(args.subdir, args.dag_id)
+    dag = _parse_and_get_dag(args.dag_id)
```

Comment:
This should just use serialized dag from the db, not reparse it on the fly like this.